spec/presenters/tslr_claim_csv_row_spec.rb

```diff
@@ -43,13 +43,35 @@
         date_of_birth,
         claim.teacher_reference_number,
         claim.national_insurance_number,
-        "Plan 2",
-        claim.email_address,
+        StudentLoans::PLAN_2.humanize,
+        '="' + claim.email_address + '"',
```

Comment:
I'm a bit worried that checking for the presence of an `@` results in a false positive like this. This means that if the CSV is opened in anything other than Excel, the email will look a bit odd. Can we not treat the email field differently? My instinct is that we _could_ trust this field anyway, given there are validations in place to only accept a valid email address (unless there's a way to craft a valid email that also acts as an Excel formula)

Reply:
I don't think this _is_ a false positive. "@" has meaning in Excel, and you might be able to embed a function into a valid email address, for instance. My understanding is that other software that consumes CSVs will also parse the extra equals sign away, so the only place this should be visible is in the raw CSV, which hopefully is only being read by developers. This is also a stop-gap that only lives as long as this CSV export does, so I think it's ok. I'd rather deal with it when there's an issue and be more complete in our sanitation.

Reply:
Yeah, good point on the stop gap bit. I'm happy in that case 👍